scripts/publish_layers.sh

```diff
@@ -85,7 +85,7 @@ publish_layer() {
     echo "Published layer for region $region, python version $aws_version_key, layer_name $layer_name, layer_version $version_nbr"
 }
 
-BATCH_SIZE=60
+BATCH_SIZE=1
```

Comment:
Is the goal here to make publishing serial by default, or to remove parallel publishing altogether? If it's the latter, I don't see a reason why we'd want to keep this extra parallel publishing code. If it's the former, do we document a way to force parallel publishing?

Reply:
The goal is to make publishing serial by default to ensure slow network connections don't cause timeout issues. If someone feels like their internet speed can handle parallel uploads then they can increase the batch size to force parallel publishing. I can document this in our internet docs as note when running `scripts/publish_prod.sh`.